pkg/sql/catalog/catprivilege/default_privilege.go

```diff
@@ -148,28 +148,35 @@ func (d *immutable) CreatePrivilegesFromDefaultPrivileges(
 	// If default privileges are not defined for the creator role, we handle
 	// it as the default case where the user has all privileges.
 	role := descpb.DefaultPrivilegesRole{Role: user}
-	if _, found := d.GetDefaultPrivilegesForRole(role); !found {
+	if defaultPrivilegesForRole, found := d.GetDefaultPrivilegesForRole(role); !found {
```

Comment:
does the comment above need adjusting?

Reply:
No this is still accurate but it might be more fitting to put this into the `if !found` clause? Thoughts?

Reply:
yeah i'm in favor of putting it inside of `if !found` and then one more comment inside of the `else`